fireplace/actions.py

```diff
@@ -194,7 +194,7 @@ def do(self, source, attacker, defender):
 		defender.defending = True
 		source.game.proposed_attacker = attacker
 		source.game.proposed_defender = defender
-		source.game.manager.step(Step.MAIN_COMBAT, Step.MAIN_ACTION)
+		source.game.manager.step(Step.MAIN_COMBAT, source.game.step)
```

Comment:
Usually nothing but should MAIN_COMBAT start from a step other than MAIN_ACTION, this will ensure that it is the previous step that is resumed afterwards.